packages/expo-image/ios/EXImageView.m

```diff
@@ -2,16 +2,39 @@
 
 #import <expo-image/EXImageView.h>
 #import <React/RCTConvert.h>
+#import <React/RCTBridge.h>
+#import <React/RCTUIManager.h>
 
 static NSString * const sourceUriKey = @"uri";
+static NSString * const sourceScaleKey = @"scale";
+static NSString * const sourceWidthKey = @"width";
+static NSString * const sourceHeightKey = @"height";
 
 @interface EXImageView ()
 
-@property (nonatomic, strong) NSURL *imageURL;
+@property (nonatomic, strong) NSDictionary *source;
+@property (nonatomic, assign) RCTResizeMode resizeMode;
+@property (nonatomic, assign) BOOL needsReload;
+@property (nonatomic, assign) CGSize intrinsicContentSize;
 
 @end
 
 @implementation EXImageView
+{
+  __weak RCTBridge *_bridge;
```

Comment:
Is there any reason for having two types of property declarations (one being with `@property`, other one with `{ properties… }`)? Does this one have any advantage over the other?

If not let's make the property declaration consistent with others:

```objc
@property (nonatomic, weak) RCTBridge *bridge;
```

Reply:
Yeah you're right that would be better. I've updated the code, thanks!